skimage/exposure/histogram_matching.py

```diff
@@ -19,7 +20,10 @@ def _match_cumulative_cdf(source, template):
     return interp_a_values[src_unique_indices].reshape(source.shape)
 
 
-def match_histograms(image, reference, *, multichannel=False):
+@utils.channel_as_last_axis(channel_arg_positions=(0, 1))
+@utils.deprecate_multichannel_kwarg()
+def match_histograms(image, reference, *, multichannel=False,
+                     channel_axis=None):
```

Comment:
Since we had the foresight to make multichannel kwarg only here, how about moving it to the final position? =)